tests/test.cpp

```diff
@@ -630,24 +427,24 @@ TEST_CASE("F functions")
                                 204, 10, 9,  10,  11, 129, 139, 171, 15, 18};
         map<uint64_t, vector<pair<Bits, Bits>>> buckets;
 
-        uint8_t k = 12;
+        uint8_t const k = 12;
         uint64_t num_buckets = (1ULL << (k + kExtraBits)) / kBC + 1;
-        Bits x = Bits(0, k);
+        uint64_t x = 0;
 
         F1Calculator f1(k, test_key_2);
         for (uint32_t j = 0; j < (1ULL << (k - 4)) + 1; j++) {
-            for (auto pair : f1.CalculateBuckets(x, 1U << 4)) {
+            for (auto pair : f1.CalculateBuckets(Bits(x, k), 1U << 4)) {
                 uint64_t bucket = std::get<0>(pair).GetValue() / kBC;
                 if (buckets.find(bucket) == buckets.end()) {
                     buckets[bucket] = vector<std::pair<Bits, Bits>>();
                 }
                 buckets[bucket].push_back(pair);
-                if (x.GetValue() + 1 > (1ULL << k) - 1) {
+                if (x + 1 > (1ULL << k) - 1) {
                     break;
                 }
                 ++x;
```

Comment:
this was the only place where the increment operator was actually used (and not just tested). However, this code just uses `Bits x` as a `uint64_t` already anyway, so I think the new version is still simpler